PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -132,10 +138,17 @@ struct SourceView: View {
             case .empty:
                 EmptyView()
             case .valid:
-                StatusBadgeView(imageName: "checkmark.circle.fill",
-                                imageColor: .green,
-                                popoverText: "preferences.popover.valid",
-                                showingPopover: $showingPopover)
+                if isEnabled {
+                    StatusBadgeView(imageName: "checkmark.circle.fill",
+                                    imageColor: .green,
+                                    popoverText: "preferences.popover.valid",
+                                    showingPopover: $showingPopover)
+                } else {
+                    StatusBadgeView(imageName: "checkmark.circle.badge.xmark.fill",
+                                    imageColor: .gray,
+                                    popoverText: "state.disabled",
```

Comment:
Same here with localisation.

Reply:
This is already in Localizable.strings; https://github.com/PlayCover/PlayCover/blob/9a66b3a5ec0bec4f181529353045022acc9d6c57/PlayCover/ca.lproj/Localizable.strings#L36